separating files

cagomezr opened the pull request:
Separating files according to issue #7

the only part I'm concerned( due to set up) is if my links to the  CSS and JavaScript are correct

they are made  as
style.css  as the  CSS files
script.js as the JavaScript scripts

please let me know.

cagomezr committed:
separating files

Separating files

jvankooo reviewed (commented):
Did you check by running it with the server? You'll have to put the files in the right directory perhaps. Please check.. cheers

cagomezr committed:
Movng scripts to static with the   other items

While it worked I think the issue I had was that you had  other scripts and wallpapers on, static, so I decided to move everything to static.

cagomezr commented:
I  did a quick test and changed  folders so it follows the location of your  static  folder, which I believe may have been why I was worried, it looks  good,  but I as a rule  don't trust my environment -I been having enviroment issues with python and react lately.

j-99 commented:
Great Job @cagomezr  !! I'll test the files and let you know. Thanks for the efforts !!
